dm/worker/status.go

```diff
@@ -194,16 +194,18 @@ func (w *Worker) Error(stName string) []*pb.SubTaskError {
 	return errs
 }
 
-// statusProcessResult returns a clone of *pb.ProcessResult, but omit the `Error` field, so no duplicated
-// error message will be displayed in `query-status`, because the `Msg` field contains enough error information.
+// statusProcessResult returns a clone of *pb.ProcessResult, but omit the `Msg` field, so no duplicated
+// error message will be displayed in `query-status`, because the `Error` field contains enough error information.
 func statusProcessResult(pr *pb.ProcessResult) *pb.ProcessResult {
 	if pr == nil {
 		return nil
 	}
 	result := proto.Clone(pr).(*pb.ProcessResult)
 	if result != nil {
 		for i := range result.Errors {
-			result.Errors[i].Error = nil
+			if result.Errors[i].Error != nil {
+				result.Errors[i].Msg = ""
```

Comment:
now, our `Msg` do not including error codes, class, etc.

update this line will change the response from 

```
                    "errors": [
                        {
                            "Type": "UnknownError",
                            "msg": "TCPReader get relay event with error: ERROR 1236 (HY000): Could not find first log file name in binary log index file",
                            "error": null
                        }
                    ],
```

to

```
                        "errors": [
                            {
                                "Type": "UnknownError",
                                "msg": "",
                                "error": {
                                    "ErrCode": 10006,
                                    "ErrClass": 1,
                                    "ErrScope": 2,
                                    "ErrLevel": 3,
                                    "Message": "run table schema failed - dbfile ./dumped_data.task_single/db_single.tbl-schema.sql: execute statement failed: CREATE TABLE `tbl` (`c1` int(11) NOT NULL,`ct` datetime NOT NULL DEFAULT '0000-00-00 00:00:00' COMMENT '创 建时间',PRIMARY KEY (`c1`)) ENGINE=InnoDB DEFAULT CHARSET=latin1;: Error 1067: Invalid default value for 'ct'",
                                    "RawCause": "Error 1067: Invalid default value for 'ct'"
                                }
                            }
                        ],
```